src/appmixer/intelswift/core/UpdateTicket/UpdateTicket.js

```
@@ -0,0 +1,21 @@
+
+const { makeRequest } = require('../../commons');
+module.exports = {
+
+    async receive(context) {
+
+        const { tenantId, projectUUID, data } = context.messages.in.content;
+
+        if(!tenantId) throw 'tenantId not found';
+        if(!projectUUID) throw 'projectUUID not found';
+        if(!data) throw 'data not found';
+        
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)